javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+setInterval(timeCode, 1000);
+
+var getMessage = document.querySelector('.countdown-display');
+var finalCountdown = 4;
+
+function timeCode() {
+  finalCountdown--;
+  if (finalCountdown > 0) {
+    getMessage.textContent = finalCountdown;
+  } else {
+    getMessage.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(finalCountdown);
```

Comment:
Even if the display no longer changes, please make sure you're actually clearing the interval.